### ISSUE TITLE
Cubian r6 hangs on mali config

### ISSUE THREAD

**Issue by seinoxygen** [cubieplayer/Cubian#196]
Cubieboard 1 with fresh install of Cubian r6 hangs after setting mali clock. I've waited for like an hour to see if it continues booting or gives me control over serial or login screen but nothing happens, it stays there.

http://pastebin.com/P6u3qwUc

**Comment by cubieplayer** [comment]
Try to rewrite the image using a lower blocksize, I am sure the image is fine.

**Comment by seinoxygen** [comment]
How to do that? I'm using Windows 7.

**Comment by cubieplayer** [comment]
Do you have a TTL cable? http://linux-sunxi.org/Cubieboard/TTL. I need the serial output to find out where is wrong.

**Comment by seinoxygen** [comment]
Yes. I pasted the output of the serial here http://pastebin.com/P6u3qwUc

**Comment by cubieplayer** [comment]
The normal boot log should continue like

> Mali device driver loaded
> [ ok ] Activating lvm and md swap...done.
> [....] Checking file systems...fsck from util-linux 2.20.1
> done.
> [ ok ] Mounting local filesystems...done.
> [ ok ] Activating swapfile swap...done.
> [ ok ] Cleaning up temporary files....

Maybe it's hanged at `Activating lvm and md swap`? I have no idea.

**Comment by seinoxygen** [comment]
Ok, i've tried it in another cubieboard and it runs without problem (same memory card and same install).

I noticed that in the first cubieboard wich hangs the led 2 stops blinking after set the mali clock.
While in the second one continues blinking after setting the mali click and then turns off when the user login appear.

Maybe its a bugged board?

**Comment by kubow** [comment]
Hello,

have you solved thiss issue? I am having the same problem. I tried Cubian r4, r5 and r6 on CubieBoard1 and all of them freezes on mali config.

Pitty is I do not have another Cubie to test it out, but with BerryBoot image on SD is everything functional.

Is there any way to get Cubian directly into NAND?
I'm really desparate, maybe I try to instal Cubian just in text mode and add other after succesfull start, I'll let you know.

**Comment by sm0svx** [comment]
I have the same problem with Cubian release >= r6. I noticed that the kernel was newer in r6 so I tried to downgrade it to the same kernel as in r5 by copying the kernel and modules from that release. Unfortunately that did not help so there is something else that differs. I still got the freeze at the _mali clock set completed, clock is  320000000 Hz_ line.

I'll do some more tests to see if I can find out what's wrong.
